### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jaimielogan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,26 +6,36 @@
...
-
+1. Core Features
+  1. Home page that lists out all happy hour deals in database
+  2. Ability to filter by day of the week
+  3. Ability to search by neighborhood
+  4. Ability to sort
+  5. Link to menu
+  6. Link to restaurant website
+2. Special Features
+  1. Map of restaurants (3rd party API)
```
Comment (BrooksPatton) [inline README.md]: Please add the technology that you will be using for the map API

**Review by BrooksPatton — CHANGES_REQUESTED** [review]
Please add what map API you will be using.

**Comment by jaimielogan** [comment]
Updated the read me! We are going to try and incorporate google maps as our 3rd party API.

**Review by BrooksPatton — APPROVED** [review]
Looks good, I can't wait to see it next week!

**Pull request closed without merging**